src/file.js

```diff
@@ -2071,22 +2120,30 @@ File.prototype.request = function(reqOpts, callback) {
 };
 
 /**
- * The Storage API allows you to use a custom key for server-side encryption.
  * This method allows you to update the encryption key associated with this
  * file.
  *
  * @see [Customer-supplied Encryption Keys]{@link https://cloud.google.com/storage/docs/encryption#customer-supplied}
  *
- * @param {string|buffer} encryptionKey An AES-256 encryption key.
+ * @param {string|buffer|object} options An AES-256 encryption key. If you'd
```

Comment:
If design is kept, should clarify that if a `string` is provided, an AES-256 encryption is expected. As it's not clear from first reading the documentation.